setupCacheEdits.ts

```diff
@@ -45,4 +48,17 @@ export function setupCacheEdits<B extends Bot>(
 
     GUILD_MEMBER_REMOVE(bot, data, shardId);
   };
+
+  bot.handlers.GUILD_UPDATE = function (_, data, shardId) {
```

Comment:
Im not a big fan of this, i think unavailable is a lot more complicated handling than just in guild update. unavailable requires handled guild deletes and creates and much more.